### ISSUE TITLE
ist it possible to include the IVEO-support?

### ISSUE THREAD

**Issue by Cockie74** [tfohlmeister/homebridge-selve#4]
I tried to run the homebridge plugin several weeks now. since I am not good at coding and also not familiar with the SELVE-System I recognized just now that my SELVE-System doesn't support Commeo but only IVEO.
Since the USB Gateway does support the IVEO I am wondering what it will take to make the plugin also working with IVEO-components...
Is it possible to include the IVEO support?

**Comment by tfohlmeister** [comment]
Hi,
yes it seems you are right: The usb gateway does support both IVEO and Commeo so this should work in theory. One problem I can see however is that IVEO is unidirectional, meaning these shutters are not able to send their current status (position, etc.) back to the gateway. While this is not a deal breaker, it could mean that your shutter positions are out of sync with what Homebridge knows, for example if you move them manually.

From what I can see in the documentation all I would need to change in the plugin is the result function. For IVEO we would just assume the new position is what we sent as a target, while for Commeo we currently wait for the shutter to report its actual position.

So I think its possible to integrate IVEO into this plugin. I don't have any IVEO hardware though, so its hard to do...

**Comment by Cockie74** [comment]
Mhhh, since I have IVEO hardware maybe it is possible that I try to test the plugin?
But as I understood the documentation of the USB Gateway the XML IVEO commands are quite different to Commeo. So maybe there are more changes necessary?

**Comment by tfohlmeister** [comment]
As far as I can tell its the same `selve.GW.command.device` command for both protocols. You can just try out the plugin with your existing setup. If I understood correctly, it should move the shutters but never update their status (so in your home app, it will stay in a loading / refreshing state forever).

**Comment by Cockie74** [comment]
No, currently it’s not moving the shutters - that’s the problem. I thought you have to use selve.GW.iveo.commandAutomatic.

**Comment by pelo76** [comment]
Cockie74 - Did you solve the problem?
